io-classes-mtl/src/Control/Monad/Class/MonadThrow/Trans.hs

```diff
@@ -0,0 +1,176 @@
+{-# LANGUAGE RankNTypes      #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Control.Monad.Class.MonadThrow.Trans () where
+
+import           Control.Monad.Except
+import           Control.Monad.RWS
+import           Control.Monad.State
+import           Control.Monad.Writer
+
+import           Control.Monad.Class.MonadThrow
+
+--
+-- ExceptT Instances
+--
+-- These all follow the @exceptions@ package to the letter
+--
+
+instance MonadCatch m => MonadThrow (ExceptT e m) where
+  throwIO = lift . throwIO
+
+instance MonadCatch m => MonadCatch (ExceptT e m) where
+  catch (ExceptT m) f = ExceptT $ catch m (runExceptT . f)
+
+  generalBracket acquire release use = ExceptT $ do
```

Comment:
The same as https://hackage.haskell.org/package/exceptions-0.10.7/docs/src/Control.Monad.Catch.html#local-6989586621679054762